modules/about-gitops.adoc

```diff
@@ -0,0 +1,12 @@
+// Module is included in the following assemblies:
+//
+// * openshift-docs/cicd/gitops/understanding-openshift-gitops.adoc
+
+[id="about-gitops_{context}"]
+= About GitOps
+
+GitOps is a declarative way to implement continuous deployment for cloud native applications. You can use GitOps to create repeatable processes for managing OpenShift Container Platform clusters and applications across multi-cluster Kubernetes environments. GitOps handles and automates complex deployments at a fast pace, saving time during deployment and release cycles. Its workflow pushes an application through all its stages, development, testing, staging, and production. GitOps either deploys a new application or updates an existing one, so you only need to update the repository. It handles and automates everything else.
```

Comment:
s/Its workflow/The GitOps workflow

Reply:
You use "staging" and "stage" in this document to refer to the same environment, you could probably standardise on "staging".